paymentsheet/src/main/java/com/stripe/android/link/account/LinkAccountUtil.kt

```diff
@@ -1,6 +1,13 @@
 package com.stripe.android.link.account
 
 import com.stripe.android.link.LinkAccountUpdate
+import com.stripe.android.model.ConsumerShippingAddress
 
 internal val LinkAccountManager.linkAccountUpdate: LinkAccountUpdate
     get() = LinkAccountUpdate.Value(linkAccount.value)
+
+internal suspend fun LinkAccountManager.loadDefaultShippingAddress(): ConsumerShippingAddress? {
+    val shippingAddresses = cachedShippingAddresses ?: listShippingAddresses().getOrNull() ?: return null
+    cachedShippingAddresses = shippingAddresses
+    return shippingAddresses.addresses.firstOrNull { it.isDefault } ?: shippingAddresses.addresses.firstOrNull()
+}
```

Comment:
something to think about in the future - there's so many ways to access link related info (via `LinkAccountManager`, via `LinkAccountHolder`, via this extension functions), sometimes with different side effects. We're following the same pattern here (where we can access shipping addresses via either `listShippingAddresses` or `loadDefaultShippingAddress`).